### PR TITLE
chat teams cannot be created with the same name

### PR DESCRIPTION
**Pull request by aslit**

### CODE REVIEW

File: group1/comm/static/comm_tool.js [hunk 1]
```diff
@@ -206,7 +206,13 @@ global.on('deletemsg', function(msgid){
 function createTeamFunc() {
 
     var new_team_name = $('input#teamname').val();
-    if(testNameValidation(new_team_name)) {
+    if(!testNameValidation(new_team_name)) {
```
Comment (gggoddardbu) [inline group1/comm/static/comm_tool.js]: There should be a space between "if" and "("

File: group1/comm/static/comm_tool.js [hunk 2]
```diff
@@ -206,7 +206,13 @@ global.on('deletemsg', function(msgid){
...
+      alert("Please enter a valid team name");
+
```
Comment (gggoddardbu) [inline group1/comm/static/comm_tool.js]: Can this newline be removed?

File: group1/comm/static/comm_tool.js [hunk 3]
```diff
@@ -206,7 +206,13 @@ global.on('deletemsg', function(msgid){
...
+    } else if(!isTeamNameExist(new_team_name)) {
```
Comment (gggoddardbu) [inline group1/comm/static/comm_tool.js]: There should be a space between "if" and "("

File: group1/comm/static/comm_tool.js [hunk 4]
```diff
@@ -206,7 +206,13 @@ global.on('deletemsg', function(msgid){
...
+      alert("A team with that name already exist.");
+
```
Comment (gggoddardbu) [inline group1/comm/static/comm_tool.js]: Can this newline be removed?

File: group1/comm/static/comm_tool.js [hunk 5]
```diff
@@ -206,7 +206,13 @@ global.on('deletemsg', function(msgid){
...
+    } else{
```
Comment (gggoddardbu) [inline group1/comm/static/comm_tool.js]: There should be a space between "else" and "{"

File: group1/comm/static/comm_tool.js [hunk 6]
```diff
@@ -592,22 +606,26 @@ function getCurrentRoom() {
 
 function editTeamFunc() {
   var edited_team_name = $('input#teamname').val();
-  if(testNameValidation(edited_team_name)) {
+  if(!testNameValidation(edited_team_name)) {
```
Comment (gggoddardbu) [inline group1/comm/static/comm_tool.js]: There should be a space between "if" and "("

File: group1/comm/static/comm_tool.js [hunk 7]
```diff
@@ -592,22 +606,26 @@ function getCurrentRoom() {
...
+    alert("Please enter a valid team name");
+
```
Comment (gggoddardbu) [inline group1/comm/static/comm_tool.js]: Can this newline be removed?

File: group1/comm/static/comm_tool.js [hunk 8]
```diff
@@ -592,22 +606,26 @@ function getCurrentRoom() {
...
+  } else if(!isTeamNameExist(edited_team_name)){
```
Comment (gggoddardbu) [inline group1/comm/static/comm_tool.js]: There should be a space between "if" and "(".  There should also be a space between ")" and "{" at the end of the line.

File: group1/comm/static/comm_tool.js [hunk 9]
```diff
@@ -592,22 +606,26 @@ function getCurrentRoom() {
...
+    alert("A team with that name already exist. ");
+
```
Comment (gggoddardbu) [inline group1/comm/static/comm_tool.js]: Can this newline be removed?

**Comment by aslit** [comment]
@gggoddardbu I changed them according to comments. Please could you review and merge it.

**Comment by gggoddardbu** [comment]
Thank you @aslit!  Looks great!

**Pull request merged**